test-it/ExpectedResults/TestVI_VariantToData_VariantWiredToDataTypeTerminal.vtr

```diff
@@ -1,2 +1,3 @@
 TestResult_Int32InAndUnwiredDataTypeInDoesNotReportError true
 TestResult_VariantOfInt32InAndUnwiredDataTypeInDoesNotReportError true
+TestResult_VariantOfInt32InWithUnwiredDataTypeInAndDataOutDoesNotReportsError true
```

Comment:
Added a new test case here to verify that if the type _and_ data terminals are unwired, we do not crash or report error.